critiquebrainz/data/model/user.py

```diff
@@ -95,9 +95,9 @@ def reviews(self):
     def avatar(self):
         """Link to user's avatar image."""
         if self.show_gravatar and self.email:
-            return "https://gravatar.com/avatar/" + hashlib.md5(self.email.encode("ascii")).hexdigest() + "?d=mm&r=pg"
+            return "https://gravatar.com/avatar/" + hashlib.md5(self.email.encode("ascii")).hexdigest() + "?d=identicon&r=pg"
         else:
-            return "https://gravatar.com/avatar/placeholder?d=mm"
+            return "https://gravatar.com/avatar/placeholder?d=identicon"
```

Comment:
Well, the whole point of using those is to have unique avatars for each user even if they don't have an email address.